src/xdocs/config_coding.xml

```diff
@@ -3623,6 +3623,200 @@ class TestHashCode {
       </subsection>
     </section>
 
+    <section name="MatchXpath">
+      <p>Since Checkstyle 8.39</p>
+      <subsection name="Description" id="MatchXpath_Description">
+        <p>
+          Evaluates Xpath query and report violation on all matching AST nodes. Current check allows
+          user to implement custom checks using Xpath. If Xpath query is not specified explicitly,
+          then check does nothing.
+        </p>
+        <p>
+          Please read more about Xpath syntax at
+          <a href="https://www.w3schools.com/xml/xpath_syntax.asp"> W3Schools Xpath Syntax</a>.
+          Information regarding Xpath functions can be found at
+          <a href="https://developer.mozilla.org/en-US/docs/Web/XPath/Functions">XSLT/XPath
+            Reference</a>. Note, that <b>@text</b> attribute can used only with token types that
```

Comment:
I noticed all these xpath links do not point to the website of the xpath utility we are using. Their website documentation doesn't have anything similar? Since we are using their utility, I figured their documentation would match more what we need.

Reply:
found similar Saxon docs, please check

Reply:
Thanks.